tests/integration/targets/grpc_config/tasks/main.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- include: junos.yaml
+  when: ansible_network_os == 'junipernetworks.junos.junos'
+  tags:
+    - netconf
+
+- include: iosxr.yaml
+  when: ansible_network_os == 'cisco.iosxr.iosxr'
+  tags:
+    - netconf
+
+- include: nxos.yaml
+  when: ansible_network_os == 'cisco.nxos.nxos'
+  tags:
+    - netconf
```

Comment:
Are we going to go back and fill these in later, or is this just going to confuse people?

I'd also rather see these tagged grpc

Reply:
This was a mistake. Fixed it.